test/openapi3/general/readOnlyWriteOnly.test.js

```diff
@@ -0,0 +1,398 @@
+'use strict';
+
+const chai = require('chai');
+const expect = chai.expect;
+const schemaValidatorGenerator = require('../../../src/index');
+const path = require('path');
+
+describe.only('oai3 - readOnly/writeOnly', function () {
```

Comment:
CI is failing because .only was committed here

Reply:
had to test that once